projects.js

```diff
@@ -0,0 +1,197 @@
+const projectsSection = document.getElementById('projects-section');
+const headerWrapper = document.querySelector('.header-container');
+const projects = [
+  {
+    name: 'Tonic',
+    jobDescription: {
+      company: 'CANOPY',
+      role: 'Back End Dev',
+      year: '2015',
+    },
+    description: {
+      featured: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+      detailed: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standar dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with thereleaLorem Ipsum is simply dummy text of the printing and typesetting industry.',
+    },
+    languages: {
+      featured: ['html', 'css', 'javascript'],
+      detailed: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    },
+    thumbnail: '1-tonic',
+  },
```

Comment:
- [ ]  You should add your link to live version and link to source code as it is structured in Figma design. This will make your website more professional, and all buttons will be working fine 
![image](https://user-images.githubusercontent.com/43877689/173362433-82690f21-6980-4549-8abd-4ee8d7aa1535.png)

![image](https://user-images.githubusercontent.com/43877689/173362991-13a58df4-68c8-4a2e-b5e3-bb07098679bd.png)